utils/src/main/scala/za/co/absa/enceladus/utils/fs/FileSystemUtils.scala

```diff
@@ -48,24 +48,5 @@ object FileSystemUtils {
         FileSystem.get(hadoopConf) // HDFS
     }
   }
-
-  implicit class FileSystemExt(fs: FileSystem) {
```

Comment:
In file `FileSystemServiceSuite.scala` there is still a comment mentioning this class

Reply:
Nice catch. Removed from there.